tools/generate-identifier-regex.js

```diff
@@ -13,7 +13,6 @@ var get = function(what) {
 // Get the Unicode properties needed to construct the regex.
 var ID_Start = get('Binary_Property/ID_Start');
 var ID_Continue = get('Binary_Property/ID_Continue');
-var Other_ID_Start = get('Binary_Property/Other_ID_Start');
```

Comment:
this was unused, but not sure if it actually _should_ be in use somewhere, in which case this will not help 😆